source/operations/external-iam/configure-keycloak-identity-management.rst

```diff
@@ -0,0 +1,267 @@
+
+This procedure assumes an existing MinIO deployment configured with Keycloak as an external identity manager.
+
+1) Create the Necessary Client Scopes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Navigate to the :guilabel:`Client scopes` view and create a new scope:
+
+.. list-table::
+   :stub-columns: 1
+   :widths: 30 70
+   :width: 100%
+
+   * - :guilabel:`Name`
+     - Set to a recognizable name for the scope (``minio-admin-API-access``)
+   * - :guilabel:`Mappers`
+     - Select :guilabel:`Configure a new mapper`
+   * - :guilabel:`Audience`
+     - Set the :guilabel:`Name` to any recognizable name for the mapping (``minio-admin-api-access-mapper``)
+   * - :guilabel:`Included Client Audience`
+     - Set to ``security-admin-console``.
+
+Navigate to :guilabel:`Clients` and select the MinIO client
+
+1. From :guilabel:`Service account roles`, select :guilabel:`Assign role` and assign the ``admin`` role
+2. From :guilabel:`Client scopes`, select :guilabel:`Add client scope` and add the previously created scope
+
+Navigate to :guilabel:`Settings` and ensure :guilabel:`Authentication flow` includes ``Service accounts roles``.
+
+2) Validate Admin API Access
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can validate the functionality by using the Admin REST API with the MinIO client credentials to retrieve a bearer token and user data:
+
+1. Retrieve the bearer token:
+
+   .. code-block:: shell
+      :class: copyable
+
+      curl -d "client_id=minio" \
+           -d "client_secret=secretvalue" \
+           -d "grant_type=password" \
+           http://keycloak-url:port/admin/realms/REALM/protocol/openid-connect/token
+
+2. Use the value returned as the ``access_token`` to access the Admin API:
+
+   .. code-block:: shell
+      :class: copyable
+
+      curl -H "Authentication: Bearer ACCESS_TOKEN_VALUE" \
+           http://keycloak-url:port/admin/realms/REALM/users/UUID
+
+   Replace the UUID with the unique ID for the user which you want to retrieve.
+   The response should resemble the following:
+
+   .. code-block:: json
+      
+      {
+         "id": "954de141-781b-4eaf-81bf-bf3751cdc5f2",
+         "createdTimestamp": 1675866684976,
+         "username": "minio-user-1",
+         "enabled": true,
+         "totp": false,
+         "emailVerified": false,
+         "firstName": "",
+         "lastName": "",
+         "attributes": {
+            "policy": [
+               "readWrite"
+            ]
+         },
+         "disableableCredentialTypes": [],
+         "requiredActions": [],
+         "notBefore": 0,
+         "access": {
+            "manageGroupMembership": true,
+            "view": true,
+            "mapRoles": true,
+            "impersonate": true,
+            "manage": true
+         }
+      }
+
+   MinIO would revoke access for an authenticated user if the returned value has ``enabled: false``, or if the Admin API returned no matching user.
```

Comment:
I'm not entirely following this paragraph. The first part makes sense: if the returned value has `enabled: false`, access is revoked even if the user would otherwise be correctly authenticated.

Does _the Admin API returned no matching user_ mean the user doesn't exist, or the user does exist but lacks some needed permission? Is there anything to revoke in that case?